client/pages/liked-songs.jsx

```diff
@@ -53,7 +56,10 @@ export default class LikedSongs extends React.Component {
     };
     fetch(`/spotify/addTracks/${this.state.playlistId}/${this.state.currentAdd}`, req)
       .then(res => res.json())
-      .catch(err => console.error(err));
+      .catch(err => {
+        console.error(err);
+        window.location.hash = '#error';
```

Comment:
Ok, instead of taking the user to a different page, you should display the error on the current page. Usually people try refreshing the page to fix something but that won't work if they're on a whole new page.